### PR TITLE
refactor(tests): clean up codes in testing `hubapi`

### PR DESCRIPTION
**Pull request by nan-wang**
refactor(tests): clean up codes in testing `hubapi`

### CODE REVIEW

**Review by JoanFM — DISMISSED** [review]

**Comment by florian-hoenicke** [comment]
I think it is failing because the access.yml is not defined right?
We could create it before running `_fetch_access_token`

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]
I think it is failing because the access.yml is not defined right?
We could create it before running _fetch_access_token

File: tests/unit/docker/test_hubapi.py [hunk 1]
```diff
@@ -52,9 +49,7 @@ def test_hubapi_list(mocker):
...
 
-@mock.patch('jina.docker.hubapi.JAML.load')
 def test_fetch_access_token(mocker):
-    token_string = json.dumps({'access_token':'dummy_token'})
-    token_json = json.loads(token_string)
-    mocker.return_value = token_json
-    _fetch_access_token(logger=getLogger())
\ No newline at end of file
+    target_val = 'dummy_token'
+    mocker.patch('jina.docker.hubapi.JAML.load', return_value={'access_token': target_val})
```
Comment (florian-hoenicke) [inline tests/unit/docker/test_hubapi.py:54]: we could insert to make sure the file exists credentials_file().touch()

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]

File: tests/unit/docker/test_hubapi.py (hunk 1 above)
Comment (nan-wang) [reply tests/unit/docker/test_hubapi.py:54]: Do you mean to insert here? We've ensured the file exists. Anyway, touching an empty one doesn't make sense in this case. 

https://github.com/jina-ai/jina/blob/master/jina/docker/hubapi.py#L250

Comment (nan-wang) [reply tests/unit/docker/test_hubapi.py:61]: ok, I see your point.

**Pull request merged**